airgun/entities/user.py

```diff
@@ -1,12 +1,13 @@
 from navmazing import NavigateToSibling
+from wait_for import wait_for
 
 from airgun.entities.base import BaseEntity
 from airgun.navigation import NavigateStep
 from airgun.navigation import navigator
 from airgun.views.user import UserCreateView
 from airgun.views.user import UserDetailsView
 from airgun.views.user import UsersView
-from wait_for import wait_for
+
```

Comment:
remove the extra space

Reply:
spoke Gaurav this is specific to precommit

Reply:
Thanks, Omkar for putting it here, all changes to this file `airgun/entities/user.py` are due to pre-commit